### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Jennifervoodoo** [Appboy/appboy-ios-sdk#341]
### Platform

iOS

### Platform Version

15.3.1

### Braze SDK Version

4.4.3

### Xcode Version

13.3.1

### Integration Method

Cocoapods

### Computer Processor

Intel

### Repro Rate

100

### Steps To Reproduce

Example:
Try to archive the project with AppBoy 4.4.3 produce this issue:
```
❌  ld: could not reparse object file in bitcode bundle: 'Invalid bitcode version (Producer: '1316.0.21.2.3_0' Reader: '1300.0.29.3_0')', using libLTO version 'LLVM version 13.0.0, (clang-1300.0.29.3)' for architecture arm64
```

### Expected Behavior

It should archive the project correctly. We don't have the issue archiving with `4.3.4`

### Actual Incorrect Behavior

Archiving project failed.

**Comment by hokstuff** [comment]
Hi @Jennifervoodoo,

Thanks for raising this and thanks @VillSkog for responding. Similar to the post above, would you be able to check to make sure that the Xcode versioning is as expected for your CI or the local environment you are using? Braze SDK version 4.4.3 was released with `Xcode 13.3.1 (13E500a)`. Please reach out if you continue to face issues!

**Comment by Jennifervoodoo** [comment]
Hi @VillSkog, @hokstuff,
Thanks a lot for your response. It was indeed the issue, I was using an older version of Xcode on my CI.
Updating the CI to use Xcode 13.3.1 fixed the issue 🙏.
